### PR TITLE
rename CNBi->CRE, add TRaaS

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

File: community/labels.yaml [hunk 1]
```diff
@@ -406,11 +401,22 @@ default:
...
       prowPlugin: label
       target: both
-    - addedBy: human
+
+    - addedBy: anyone
       color: 0052cc
       description:
-        Issues or PRs related to the Custom Notebook Image ODH feature.
-      name: wg/cnbi
+        Issues or PRs related to the Custom Runtime Environment (fka Custom Notebook Image)
```
Comment (VannTen) [inline community/labels.yaml]:
```suggestion
        Issues or PRs related to the Custom Runtime Environment
```
The information is already there in `previously` (I'm not sure where the description of labels is shown in gh UI anyway ?)

**Review by VannTen — CHANGES_REQUESTED** [review]
One small thing, lgtm otherwise

File: community/labels.yaml (hunk 1 above)
Comment (codificat) [reply community/labels.yaml]: > I'm not sure where the description of labels is shown in gh UI anyway

It is: https://github.com/thoth-station/meteor-operator/labels?q=wg
but the `previously` references aren't.

So, I think it makes sense to leave that comment in the description for the time being

**Pull request merged**